noxfile.py

```diff
@@ -5,7 +5,7 @@
 
 import nox
 
-nox.options.reuse_existing_virtualenvs = False
+nox.options.reuse_existing_virtualenvs = True
```

Comment:
```suggestion
nox.options.reuse_existing_virtualenvs = False
```
mistakenly committed this. we want these to rebuild but I've been running tests over and over so there's no point in rebuilding  every time